### PR TITLE
{tools}[GCCcore/12.2.0] OpenStackClient v6.0.0 w/ Python 3.10.8

### PR DESCRIPTION
**Pull request by SebastianAchilles**
(created using `eb --new-pr`)

Depends on:
- https://github.com/easybuilders/easybuild-easyconfigs/pull/16676

### CODE REVIEW

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
zen2-ubuntu-2204 - Linux Ubuntu 22.04.1 LTS (Jammy Jellyfish), x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.10.6
See https://gist.github.com/de09e1603559353fc6ac50be3396ac81 for a full test report.

File: easybuild/easyconfigs/o/OpenStackClient/OpenStackClient-6.0.0-GCCcore-12.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+exts_default_options = {
+    'use_pip': True,
+}
+exts_list = [
+    ('pyperclip', '1.8.2', {
+        'checksums': ['105254a8b04934f0bc84e9c24eb360a591aaf6535c9def5f29d92af107a9bf57'],
+    }),
+    ('importlib_metadata', '5.0.0', {
+        'checksums': ['da31db32b304314d044d3c12c79bd59e307889b287ad12ff387b3500835fc2ab'],
+    }),
```
Comment (branfosj) [inline easybuild/easyconfigs/o/OpenStackClient/OpenStackClient-6.0.0-GCCcore-12.2.0.eb:27]:
```suggestion
    ('importlib_metadata', '5.0.0', {
        'checksums': ['da31db32b304314d044d3c12c79bd59e307889b287ad12ff387b3500835fc2ab'],
    }),
```

https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/p/Python/Python-3.10.8-GCCcore-12.2.0.eb#L191

**Review by branfosj — CHANGES_REQUESTED** [review]

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/o/OpenStackClient/OpenStackClient-6.0.0-GCCcore-12.2.0.eb (hunk 1 above)
Comment (SebastianAchilles) [reply easybuild/easyconfigs/o/OpenStackClient/OpenStackClient-6.0.0-GCCcore-12.2.0.eb]: Oh, I didn't noticed this. Thank you! Done in https://github.com/easybuilders/easybuild-easyconfigs/pull/16677/commits/cc7a26bef21b44ae8a6ce40df5f02f6b81e23a13

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen2

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0105u36b.bear.cluster - Linux RHEL 8.5, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/ccf656b45ca0b8ba22a193ba1df2f9c5 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux Ubuntu 20.04.2 LTS (Focal Fossa), x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/8786594d1a187c12473ec215a8bf0689 for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @SebastianAchilles!

**Pull request merged**